tests/gnrc_netif/tests/01-run.py

```diff
@@ -20,49 +20,39 @@ def testfunc(child):
     # output cross-checked hex data with WireShark -> "Import from Hex Dump..."
     # test_netapi_send__raw_unicast_ethernet_packet
     child.expect("Sending data from Ethernet device:")
-    child.expect("00000000  3E  E6  B5  0F  19  23  3E  E6  B5  22  FD  0A  " +
-                           "FF  FF  41  42")
+    child.expect("00000000  3E  E6  B5  0F  19  23  3E  E6  B5  22  FD  0A  FF  FF  41  42")
```

Comment:
Doesn't this also clash with another [PEP8 rule](https://www.python.org/dev/peps/pep-0008/#maximum-line-length)?

Reply:
Yes by default we use 80 max line length but since it's not very handy with tests scripts, the convention in #8141 proposes to increase this to 120.

Reply:
Okay.